pkg/cache/cluster.go

```diff
@@ -730,7 +730,7 @@ func (c *clusterCache) sync() error {
 		if !ok {
 			return err
 		}
-		c.log.Error(e, "warning loading openapi schema")
+		c.log.Info(e, "warning loading openapi schema")
```

Comment:
Please replace this line with:
`c.log.Info("warning loading openapi schema: %s", e)`